interchain-security/dungeon-1/README.md

```diff
@@ -0,0 +1,78 @@
+
+# `dungeon` Chain Details
+
+> Status: **PENDING**
+
+The `dungeon` chain is launched to test the dungeon chain.
+
+---
+
+<!--
+  *Compress the dungeond file (due to size)*
+  cp $(which dungeond) ./dungeond-linux-amd64
+  shasum -a 256 dungeond-linux-amd64
+  tar -czvf dungeond-linux-amd64.tar.gz dungeond-linux-amd64
+  rm ./dungeond-linux-amd64
+
+  *Get CCV Only from genesis*
+  cat genesis.json | jq .app_state.ccvconsumer > dungeon-ccv.json
+
+  *Genesis Without CCV Consumer*
+  cp genesis.json tmp-without-ccv.json
+  jq 'del(.app_state.ccvconsumer)' tmp-without-ccv.json > dungeon-genesis-without-ccv.json
+  rm tmp-without-ccv.json genesis.json
+
+  *Compress Dungeon Gneisis Without CCV*
+  tar -czvf dungeon-genesis-without-ccv.json dungeon-genesis-without-ccv.json
+-->
+
+
+* **Chain-ID**: `dungeon-1`
+* **denom**: `udgn`
+* **Spawn time**: `2024-08-28T13:30:00.000000Z`
+* **Stop time**: `-`
+* **GitHub repo**: [Crypto-Dungeon/dungeonchain](https://github.com/Crypto-Dungeon/dungeonchain)
+* **Commit**: [`f2325ed0e0d341f4280992ac3cbc61b9a967d578`](https://github.com/Crypto-Dungeon/dungeonchain/commit/f2325ed0e0d341f4280992ac3cbc61b9a967d578)
+* **Genesis file with CCV state:** [genesis.json.tar.gz](genesis.json.tar.gz)
+  * `tar -xvzf genesis.json.tar.gz "./network/dungeon-1/genesis.json" --one-top-level=genesis.json --strip-components 4`
```

Comment:
You can leave the "Genesis file with CCV state" as TBA: there is no CCV until after the spawn time is reached.

Reply:
5e7cb4f3e42f9a57bcc51c2395a39fc019f13a50